packages/kbn-unified-data-table/src/components/data_table_columns.tsx

```diff
@@ -155,7 +155,7 @@ function buildEuiGridColumn({
       ],
     },
     cellActions,
-    visibleCellActions,
+    visibleCellActions: 3,
```

Comment:
Why did you made this change? I don't think is needed. Lens doesn't use this component and having this variable set to 3 seems wrong to me.